Platforms/QemuQ35Pkg/QemuQ35Pkg.dsc

```diff
@@ -1105,6 +1110,41 @@ PlatformSmmProtectionsTestLib|UefiTestingPkg/Library/PlatformSmmProtectionsTestL
   SecurityPkg/Tcg/Tcg2Config/Tcg2ConfigDxe.inf
 !endif
 
+  #
+  # PRM Configuration Driver
+  #
+  PrmPkg/PrmConfigDxe/PrmConfigDxe.inf {
+    <LibraryClasses>
+      NULL|PrmPkg/Samples/PrmSampleAcpiParameterBufferModule/Library/DxeAcpiParameterBufferModuleConfigLib/DxeAcpiParameterBufferModuleConfigLib.inf
+      NULL|PrmPkg/Samples/PrmSampleContextBufferModule/Library/DxeContextBufferModuleConfigLib/DxeContextBufferModuleConfigLib.inf
+      NULL|PrmPkg/Samples/PrmSampleHardwareAccessModule/Library/DxeHardwareAccessModuleConfigLib/DxeHardwareAccessModuleConfigLib.inf
+  }
+
+  #
+  # PRM Module Loader Driver
+  #
+  PrmPkg/PrmLoaderDxe/PrmLoaderDxe.inf
+
+  #
+  # PRM SSDT Installation Driver
+  #
+  PrmPkg/PrmSsdtInstallDxe/PrmSsdtInstallDxe.inf
+
+  #
+  # PRM Sample Modules

```

Comment:
Real nit picky thing, what do you think about doing comments on one line instead of adding extra lines around the comments. It really expands the DSC and makes it a lot longer. I'm fine if you want to leave it, since it does draw the eye a bit better

Reply:
I think that makes sense since this just describe individual items they don't need to draw attention quite as strongly.

Reply:
Updated in commit https://github.com/microsoft/mu_tiano_platforms/pull/5/commits/2f3243ab2f3f1485a76f7ca25bb4321e20e13c3c